src/components/Notifier/index.tsx

```diff
@@ -1,63 +1,65 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { useSnackbar } from 'notistack';
 import {
   createStyles,
-  IconButton,
+  Snackbar,
   WithStyles,
   withStyles,
 } from '@material-ui/core';
-import CloseIcon from '@material-ui/icons/Close';
+import { Alert } from '@material-ui/lab';
 import {
+  Notification,
   notificationsSelector,
   removeNotification,
   setAsDisplayed,
 } from '../../context/notificationStateSlice';
 
-// built with the help of https://codesandbox.io/s/github/iamhosseindhv/notistack/tree/master/examples/redux-example?file=/Notifier.js
 function Notifier({ classes }: NotifierProps) {
   const dispatch = useDispatch();
-  const { enqueueSnackbar, closeSnackbar } = useSnackbar();
   const notifications = useSelector(notificationsSelector);
   const autoHideDuration = 50000;
+  const [openNotification, setOpenNotification] = useState<Notification | null>(
+    null,
+  );
```

Comment:
Why do we need to keep track of `openNotification` here in local state? Why can't we just have this component pull the array of notifications from Redux and render them (or render `null` if the array is empty). Then we can still get stacking easily, and just dispatch any dismiss actions.